index.html

```diff
@@ -59,150 +77,168 @@
-                    </a>
-                  </div>
+                <li class="carousel-food-spin__item js-carousel-food-spin-items">
+                  <dishe-plato image="food1.png" price="66" title="A Healthful Winter Salad" description="A mix of winter vegetables and fruits, beets, citrus, and roasted nuts!" ellipseColor="#e8f4ee" shadowColor= "#c4e6d7" buttonColor="#4fd299" hoverColor="#019a57"></dishe-plato>
                 </li>
-                <li class="carousel-food-spin__item">
-                  <div class="carousel-food-spin__dot">
-                    <a href="" aria-label="click to see the menus of dish 4">
-                      <img class="carousel-food-spin__item-img" src="image/food4.png" alt="dish 4">
-                    </a>
-                  </div>
+                <li class="carousel-food-spin__item js-carousel-food-spin-items">
+                  <dishe-plato image="food4.png" price="50" title="Shrimp, Cucumber and Avocado Salad" description="Healthy, vegetarian, protein-rich, easy to make, and affordable." ellipseColor="#9ca3ff" shadowColor= "#b6b2f3" buttonColor="#4a4dff" hoverColor="#04067b"></dishe-plato>
                 </li>
               </ul>
             </div>
-            <button class="carousel-food-spin__prev js-prev" aria-label="see plates on the right">
-              <svg width="14" height="14" viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg" aria-hidden="true">
-                <path d="M7.00574 1.00001L0.999699 6.99396M7.00574 1.00001L12.9997 7.00605M7.00574 1.00001L6.99365 13" stroke="white" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
-              </svg>
+            <div class="carousel-food-spin__prev-next">
+              <button class="carousel-food-spin__prev js-prev" aria-label="see plates on the right">
+                <svg width="14" height="14" viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg" aria-hidden="true">
+                  <path d="M7.00574 1.00001L0.999699 6.99396M7.00574 1.00001L12.9997 7.00605M7.00574 1.00001L6.99365 13" stroke="white" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+                </svg>
+              </button>
+              <button class="carousel-food-spin__next js-next" aria-label="see plates on the left">
+                <svg width="14" height="14" viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg" aria-hidden="true">
+                  <path d="M7 13L13 7M7 13L1 7M7 13V1" stroke="white" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+                </svg>
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="ventana-modal js-ventana-Modal">
+      <div class="ventana-modal__content">
+        <div class="ventana-modal__image">
+          <img class="ventana-modal__img" src="image/food5.png" alt="">
+        </div>
```

Comment:
No hay ningun estilo asociado a `ventana-modal__image` e incluso sigue funcionando igual sin ese div
```suggestion
        <img class="ventana-modal__img" src="image/food5.png" alt="">
```